docs/how_to_add_packages.md

```diff
@@ -1,8 +1,11 @@
 # Adding Packages to ConanCenter
 
-The [conan-center-index](https://github.com/conan-io/conan-center-index) (this repository) contains recipes for the remote [JFrog ConanCenter](https://conan.io/center/).
-This remote is added by default to a clean installation of the Conan client. Recipes are contributed by opening pull requests as explained in the sections below.
-When pull requests are merged, the CI will upload the generated packages to the [conancenter](https://conan.io/center/) remote.
+ConanCenter aims to provide the best quality packages for any open source project.
+Any C/C++ project can be made available by contributing a "recipe".
+
+Getting started is easy. Try building an existing package with our [developing recipes](../developing_recipes_locally.md) tutorial.
+To deepen you understanding, start with the [How to provide a good recipe](#how-to-provide-a-good-recipe) section.
+You can follow the three steps (:one: :two: :three:) described below! :tada:
```

Comment:
Should not be 1, 2, 3 links to sections in this doc?

Reply:
Brilliant idea! sadly this one does not break like that but


https://github.com/prince-chrismc/conan-center-index/commit/bb59e8ad51800bdeaf87ca2ed35ad7c639b7ab7a

🤞 should be able to fix that soon